### PR TITLE
Recomended jobs

### PR DESCRIPTION
**Pull request by amrathmd**
image
image
image

### CODE REVIEW

File: src/pages/Home/Home.tsx [hunk 1]
```diff
@@ -62,6 +96,31 @@ const Home = () => {
...
+                  {element.company.name}
+                </h1>
+              </div>
+            ))}
+          
+        
+          </Slider>
+          <div style={{ textAlign: 'center' }}>
+            <button className="prevButton" onClick={previous}>
+          &lt;
```
Comment (maharjunm) [inline src/pages/Home/Home.tsx]: what are these characters?

File: src/pages/Home/Home.tsx [hunk 2]
```diff
@@ -62,6 +96,31 @@ const Home = () => {
...
+            </button>
+            <button className="nextButton" onClick={next}>
+          &gt;
```
Comment (maharjunm) [inline src/pages/Home/Home.tsx]: here also

File: .DS_Store
Comment (maharjunm) [inline .DS_Store:1]: please remove this file.

**Comment by maharjunm** [comment]
please add screenshots

**Comment by amrathmd** [comment]
Screenshot 2023-04-17 at 5 44 52 PM

**Comment by amrathmd** [comment]
If no Job exist the component is not displayed if companies are greater than or equal to 3 three companies are shown and if less than 3 only 1/1 is shown

**Comment by amrathmd** [comment]
Screenshot 2023-04-18 at 1 20 43 PM

**Comment by amrathmd** [comment]
Screenshot 2023-04-18 at 3 12 15 PM

**Review by maharjunm — APPROVED** [review]

**Pull request merged**